themes/bitcore/layout/index.ejs

```diff
@@ -9,6 +9,7 @@
   <div class="outer">
     <div class="inner">
       <h2 id="banner-title">A pure and powerful JavaScript Bitcoin library.</h2>
+      <h2 id="banner-subtitle">Take advatage of the blockchain with a clean and simple API</h2>
```

Comment:
We should mention "Bitcoin" and "JavaScript" and "library" here, what's the reasoning for removing those?